packer/scripts/al2/al2-agent-setups.sh

```diff
@@ -37,6 +37,10 @@ sudo usermod -a -G docker $CURR_USER
 sudo yum-config-manager --add-repo https://cli.github.com/packages/rpm/gh-cli.repo
 sudo yum install -y gh
 
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
+. ~/.nvm/nvm.sh && nvm install 16
+npm install -g fs-extra chalk@4.1.2 @aws-cdk/cloudformation-diff aws-cdk cdk-assume-role-credential-plugin@1.4.0
```

Comment:
Have offline discussion with @rishabh6788 and his code will take care of the `nvm use`.
Requesting a comment on why these packages are needed for the cluster cdk.
Thanks.